SpatialGDK/Source/SpatialGDK/Public/SpatialView/OpList/StringStorage.h

```diff
@@ -10,6 +10,7 @@
 
 namespace SpatialGDK
 {
+/** Stores a utf8 string. */
```

Comment:
from previous submits, but constructors should be explicit, and class marked as final imho.

Reply:
In this case it's deliberately not marked with explicit as this is something you likely want to just implicitly convert. We can make it not do that but it would seem weird for a string storage type.

As for marked as final we can do although I don't think that's the style used anywhere else, and I haven't ever seen an error caused by that.